src/pages/NotFound.js

```diff
@@ -0,0 +1,73 @@
+import { NavLink } from 'react-router-dom';
+import { isChrome } from 'react-device-detect';
+import { GoHome } from 'react-icons/go';
+import { GoChevronLeft } from 'react-icons/go';
+import styled from 'styled-components';
+import { keyframes } from 'styled-components';
```

Comment:
I'd suggest combining imports here.

```suggestion
import { GoHome, GoChevronLeft } from 'react-icons/go';
import styled, { keyframes } from 'styled-components';
```

Reply:
Thanks for the hint 🙂